nncf/common/pruning/export_helpers.py

```diff
@@ -50,3 +61,192 @@ def get_all_op_aliases(cls):
             op_types.extend(subtype.get_all_aliases())
         op_types = list(set(op_types)) + cls.additional_types
         return op_types
+
+
+class OpInput(DefaultMetaOp):
```

Comment:
I think we really should introduce naming-level separation of these additional entities (that we use for mask propagation and operation marking for the pruning algo specifically) from the generic metatypes that we use during NNCF graph preparation. If possible, the word `Meta` should not be used in the `DefaultMetaOp` name and basically anywhere where we do not handle in actual `OperatorMetatype`s. Vice versa - everywhere we actually refer to `OperatorMetatype` names in the pruning context, such as when mapping the generic `OperatorMetatype` to the pruning-specific markers, we should use the names `meta`-something. The `subtypes` field in PT pruning markers, for instance, should be renamed to `associated_metatypes` or a similar, more specific identifier.

Reply:
Done, I've  renamed all pruning operations to `TF/PTOpNamePruningOp`